tests/unit/ForeignJurisdiction.spec.ts

```diff
@@ -29,80 +29,124 @@ describe('ForeignJurisdiction', () => {
+    expect(vm.selectedRegion).toEqual(JSON.stringify({code: 'ON', name: 'Ontario'}))
+    wrapper.destroy()
+  })
+
+  it('no BC as a province when Canada is selected as the country', async () => {
+    const wrapper = mount(ForeignJurisdiction,
+      {
+        vuetify
+      })
 
-    await Vue.nextTick()
-    wrapper.find('#region-selector').trigger('click')
-    wrapper.find('.menuable__content__active').findAll('.v-list-item').at(7).trigger('click')
-    await Vue.nextTick()
-    expect(vm.selectedRegion).toEqual({name: 'Ontario', short: 'ON'})
+    await wrapper.find('#country-selector').trigger('click')
+    await wrapper.findAll('.v-list-item').at(39).trigger('click')
+
+    const provinces = wrapper.findAll('.v-select').at(1).props('items')
+
+    expect(!provinces.includes({'name': 'British Columbia', 'short': 'BC'}))
     wrapper.destroy()
   })
 
-  xit('emit correct validation when country is selected', async () => {
+  it('Federal exists as a province when Canada is selected as the country', async () => {
+    const wrapper = mount(ForeignJurisdiction,
+      {
+        vuetify
+      })
+
+    await wrapper.find('#country-selector').trigger('click')
+    await wrapper.findAll('.v-list-item').at(39).trigger('click')
+
+    const provinces = wrapper.findAll('.v-select').at(1).props('items')
+
+    expect(provinces.includes({'name': 'Federal', 'short': 'FD'}))
+    wrapper.destroy()
+  })
+
+  it('set correct US state when selected via the region selector', async () => {
     const wrapper = mount(ForeignJurisdiction,
       {
         vuetify
       })
     const vm: any = wrapper.vm
 
-    const countriesSelector = wrapper.find('#country-selector')
-    await countriesSelector.setValue('Lebanon')
+    await wrapper.find('#country-selector').trigger('click')
+    await wrapper.findAll('.v-list-item').at(235).trigger('click')
 
-    vm.emitChangedCountry()
+    const region = wrapper.find('#region-selector')
+    await region.setValue(JSON.stringify({code: 'NY', name: 'New York'}))
 
-    expect(wrapper.emitted('valid')).toBeTruthy()
+    expect(vm.selectedCountry).toEqual({code: 'US', name: 'United States of America'})
+    expect(vm.selectedRegion).toEqual(JSON.stringify({code: 'NY', name: 'New York'}))
     wrapper.destroy()
   })
 
-  xit('emit correct validation when country is selected but region is not', async () => {
+  it('emit correct events when country is selected', async () => {
     const wrapper = mount(ForeignJurisdiction,
       {
         vuetify
       })
 
-    const countriesSelector = wrapper.find('#country-selector')
-    await countriesSelector.setValue('Canada')
+    await wrapper.find('#country-selector').trigger('click')
+    wrapper.findAll('.v-list-item').at(123).trigger('click')
 
-    expect(wrapper.emitted('valid')).toBeFalsy()
+    expect(wrapper.emitted('update:country').pop()[0]).toEqual({code: 'LB', name: 'Lebanon'})
+    expect(wrapper.emitted('valid')).toBeTruthy()
```

Comment:
I recommend using `toBe(true)` (and false in test below) so that we check explicity values. This prevents errors where, for instance, `{}` is truthy. And so is `123`. But neither of those are correct for "valid".